style.css

```diff
@@ -0,0 +1,851 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.underline3,
+hr {
+  width: 60px;
+}
+
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] Kindly ensure your design is **mobile-first**, media queries are used to change the design from mobile to desktop, not the other way around.